### ISSUE TITLE
Edit profile: Tooltips not available to all users and not dismissible/hoverable

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#39]
URL: https://hcdev.openplus.ca/en/user/203/edit
User Agent: Chrome, Firefox
Language: both

Issue Description: Tooltips must be dismissible, hoverable and available to all users (keyboard, mouse, screen reader). Similar to issue #27, there are a few issues with this. There are several tooltips on this page.

Issue 1: There is no mechanism provided to dismiss the tooltips. For example, using the mouse to hover on Email address, tool tip appears. User is not able to dismiss this tooltip.
Expected result Issue 1: Provide a mechanism to easily dismiss the additional content, such as by pressing Escape.

Issue 2: Tooltip must be available to keyboard users as well. When tabbing with keyboard, user is not able to see these tooltips and most of them have important information.
Expected result issue 2: information available for mouse users should be equally available to those using keyboard only.
Success criterion: https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

The information that is in the tooltips should be part of the page at all times. Below are some examples of tooltips that have important information that is not available to all users. 

For example, For Email address, tooltip has information for the user which disappears as user moves away the mouse.

![image](https://user-images.githubusercontent.com/14280062/128408385-44b1d842-e7dd-4f13-b534-96330e7456d1.png)

User name has important instructions

![image](https://user-images.githubusercontent.com/14280062/128408522-dcfe44f7-1a6c-4266-8d8f-cd54981dfd46.png)

Password field has tooltip covering important text behind it

![image](https://user-images.githubusercontent.com/14280062/128408676-e524bf43-5a76-4899-ae9f-37ea4cc59d46.png)

**Comment by davidgotlieb** [comment]
﻿no longer an issue, tooltips have been disabled as per mmiron-HC recommendation at ﻿https://github.com/mmiron-HC/WST-RSAMS/issues/69

**Comment by qanitah** [comment]
The instructions are visible now and aria-describedby is used to announce it with the field
